Cargo.toml

```diff
@@ -15,3 +15,4 @@ documentation = "https://docs.rs/sendfd"
 
 [dependencies]
 libc = "0.2"
+tokio = { version = "0.2", features = [ "uds" ], optional = true }
```

Comment:
May want to name this in some way that makes it obvious this is 0.2, given that 0.3 is now the "current".